lms/templates/learner_dashboard/course_enroll.underscore

```diff
@@ -1,4 +1,9 @@
-<% if (is_enrolled && (typeof expired === 'undefined' || expired === false)) { %>
+<% if (is_enrolled && collection_course_status === 'completed') { %>
+    <div class="final-grade">
+        <div class="grade-header"><%- gettext('Final Grade') %></div>
```

Comment:
It would be great if we could add some screen reader context text here to note the course name, something like ```<span class="sr"> for Course Name</span>```.